backend/app/tests/api/routes/collections/test_collection_list.py

```
@@ -0,0 +1,130 @@
+from sqlalchemy.orm import Session
+
+from app.core.config import settings
+from app.tests.utils.utils import get_project
+from app.tests.utils.collection import (
+    get_collection,
+    get_vector_store_collection,
+)
+
+
+def test_list_collections_returns_api_response(
+    db: Session,
```

Ruff (0.14.3)

Unused function argument: `db`

(ARG001)